pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -723,11 +723,11 @@ func waitUntilDNSReady(zedagentCtx *zedagentContext, stillRunning *time.Ticker)
 			dnsCtx.subDeviceNetworkStatus.ProcessChange(change)
 			if dnsCtx.triggerHandleDeferred {
 				// Connectivity has been restored so kick the queue
-				// in order to process all deferred requests
-				// immediately. We don't bother to kick the periodic
-				// queue, because failed requests will be dropped
-				// from the queue.
-				zedcloudCtx.DeferredEventCtx.KickTimer()
+				// in order to process all deferred requests faster,
```

Comment:
> in order to process all deferred requests faster
Without this kick, what would trigger the retry otherwise and how long would it take? (just asking)

Reply:
up to 15 minutes, depends on down time